extensionsGallery-insider.json

```diff
@@ -3183,6 +3183,55 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "65",
+					"extensionName": "sql-prompt",
+					"displayName": "Redgate SQL Prompt",
+					"shortDescription": "Write and format your SQL effortlessly",
+					"publisher": {
+						"displayName": "Redgate",
+						"publisherId": "Redgate",
+						"publisherName": "Redgate"
+					},
+					"versions": [
+						{
+							"version": "0.2.1",
+							"lastUpdated": "05/15/2020",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
+									"source": "https://download.red-gate.com/EAP/SQLPromptADS.zip"
```

Comment:
@twalsh92 one note is that typically this would refer to a landing page that has an option to download the VSIX.  I guess here the user would get a ZIP file directly that contains the VSIX.  Is this the desired experience?

Reply:
I guess we are ok with this right now. We can create a separate PR if needed in the future. Thanks